test/sendMessage.test.js

```diff
@@ -0,0 +1,147 @@
+const chai = require('chai');
```

Comment:
These tests already existed in `src/service/emailService.test.js`. I just moved them to their own file. You can ignore this file in your review.